test/tests.json

```diff
@@ -606,5 +606,23 @@
       "compact"
     ],
     "workdir": "rs/bin"
+  },
+  {
+    "name": "secure",
+    "client": {
+      "command": [
+        "test_secure.bash"
+      ]
+    },
+    "transports": [
+      "buffered"
+    ],
+    "sockets": [
+      "ip-ssl"
+    ],
+    "protocols": [
+      "binary"
+    ],
+    "workdir": "secure"
```

Comment:
Simply moving the same JSON entry (with relative file path adjustment) might get this included to `make crossfeature`.
In that case, known_failure file is also in `features/known_...`.

Otherwise we can leave it as is and plan to relocate later.
I guess I'll need to put some information to `test/README.md` or create `test/features/README.md`